### ISSUE TITLE
Changing path of config.js

### ISSUE THREAD

**Issue by jameswyse** [typhonjs/esdoc-plugin-jspm#1]
If you change the path to `config.js` in JSPM then this plugin fails to find it:

```
esdoc-plugin-jspm - Error: could not locate JSPM / SystemJS 'config.js'.
```

The path is set in `package.json` under `jspm.configFile` so that's probably a good place to read it from, otherwise a config option would do the job.

**Comment by typhonrt** [comment]
Thanks, I'll take a look at this soon. Indeed all the TyphonJS repos are storing the default `config.js` in the root path.

**Comment by typhonrt** [comment]
This issue should be fixed with `esdoc-plugin-jspm@0.5.0`. The NPM module is updated along with `typhonjs-core-gulptasks@0.4.0`. 

It should be noted that a standalone NPM module [typhonjs-config-jspm-parse](https://github.com/typhonjs/typhonjs-config-jspm-parse) was created from the initial JSPM / SystemJS parsing code in `esdoc-plugin-jspm` with a major difference that it parses the JSPM runtime versus the config file directly. In the latest release of `esdoc-plugin-jspm` the parsing is handled by `typhonjs-config-jspm-parse` and should pick up all JSPM configuration data from `package.json`.  

Let me know if the problem isn't fixed for you of course. Thanks for the report!
